tests/lib/rules/no-misleading-character-class.js

```diff
@@ -628,6 +628,38 @@ ruleTester.run("no-misleading-character-class", rule, {
                 suggestions: null
             }]
         },
+        {
+            code: String.raw`/[\ud83d\u{dc4d}]/u`,
+            env: { es2020: true },
```

Comment:
```suggestion
```

This isn't necessary for this test case, so it might be confusing why it is here.